src/App/Pages/Accounts/LoginPasswordlessRequestViewModel.cs

```diff
@@ -221,6 +221,13 @@ private async Task CheckLoginRequestStatus()
                 var authResult = await _authService.LogInPasswordlessAsync(Email, _requestAccessCode, _requestId, _requestKeyPair.Item2, response.Key, response.MasterPasswordHash);
                 await AppHelpers.ResetInvalidUnlockAttemptsAsync();
 
+                if(authResult == null && await _stateService.IsAuthenticatedAsync())
```

Comment:
⛏️ Add space after the `if`, I think this should have been caught by the formatter:
```suggestion
                if (authResult == null && await _stateService.IsAuthenticatedAsync())
```